stratum/proto/temp/CMakeLists.txt

```diff
@@ -0,0 +1,399 @@
+# Builds protobuf object libraries
+
+cmake_minimum_required(VERSION 3.5)
+
+find_package(Protobuf REQUIRED)
+if(NOT Protobuf_FOUND)
+    message(SEND_ERROR "Error: Protobuf package not found")
+endif()
+
+###############################
+# Get path to GRPC C++ plugin #
+###############################
+
+execute_process(
+    COMMAND which grpc_cpp_plugin
+    OUTPUT_VARIABLE GRPC_CPP_PLUGIN
+    RESULT_VARIABLE GRPC_CPP_PLUGIN_ERROR
+)
+
+if(GRPC_CPP_PLUGIN_ERROR)
+    message(FATAL_ERROR "Error: Could not find grpc_cpp_plugin")
+else()
+    string(STRIP "${GRPC_CPP_PLUGIN}" GRPC_CPP_PLUGIN)
+endif()
+
+##########################
+# Compile protobuf files #
+##########################
+
+set(PROTO_PARENT_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+
+# List of directories to be searched for Protobuf inputs.
+string(JOIN ":" PROTO_IMPORT_PATH
+    ${PROTO_PARENT_DIR}
+    ${GOOGLE_SOURCE_DIR}
+    ${P4RUNTIME_SOURCE_DIR}/proto
+    ${STRATUM_SOURCE_DIR}
+    /usr/local/include
+)
+
+set(RPC_PROTO_FILES
+    google/rpc/status.proto
+    google/rpc/code.proto
+)
+
+set(P4V1_PROTO_FILES
+    p4/v1/p4data.proto
+    p4/v1/p4runtime.proto
+    p4/config/v1/p4info.proto
+    p4/config/v1/p4types.proto
+)
+
+set(STRATUM_P4_PROTO_FILES
+    stratum/public/proto/error.proto
+    stratum/hal/lib/common/common.proto
+    stratum/hal/lib/p4/forwarding_pipeline_configs.proto
+    stratum/hal/lib/phal/db.proto
+)
+
+set(GNMI_PROTO_FILES
+    gnmi/gnmi.proto
+    gnmi/gnmi_ext.proto
+)
+
+set(YGOT_PROTO_PATH github.com/openconfig/ygot/proto)
+
+set(YGOT_PROTO_FILES
+    ${YGOT_PROTO_PATH}/yext/yext.proto
+    ${YGOT_PROTO_PATH}/ywrapper/ywrapper.proto
+)
+
+set(OPENCONFIG_PROTO_FILES
+    openconfig/enums/enums.proto
+    openconfig/openconfig.proto
+)
+
+set(STRATUM_OC_PROTO_FILES
+    stratum/public/proto/openconfig-goog-bcm.proto
+)
+
+set(STRATUM_BF_PROTO_FILES
+    stratum/public/proto/p4_table_defs.proto
+    stratum/public/proto/p4_annotation.proto
+    stratum/hal/lib/p4/p4_control.proto
+    stratum/hal/lib/p4/common_flow_entry.proto
+    stratum/hal/lib/p4/p4_table_map.proto
+    stratum/hal/lib/p4/p4_pipeline_config.proto
+    stratum/hal/lib/tdi/tdi.proto
+)
+
+########################
+# generate_proto_files #
```

Comment:
We are also generating C++ files for all proto files in stratum/proto/CMakeLists.txt (See line 92 of this file). Do you mean to remove it from there as now it's handled here?

Reply:
I committed this file by accident. I have deleted it from the PR.